src/zenml/orchestrators/utils.py

```diff
@@ -113,6 +115,17 @@ def get_config_environment_vars(
             # If a schedule is given, this is a long running pipeline that
             # should not have an API token that expires.
             expires_minutes = None
+            logger.warning(
```

Comment:
@stefannica maybe I'm missing something, but we show this warning message even if the user is currently authenticated with device/SA, no?

Reply:
Not after merging in my other PR (zenml login).